relate/urls.py

```diff
@@ -391,6 +391,15 @@
         "/$",
         course.flow.finish_flow_session_view,
         name="relate-finish_flow_session_view"),
+    url(r"^course"
+        "/" + COURSE_ID_REGEX +
+        "/flow-session"
+        "/(?P<flow_session_id>[0-9]+)"
+        "/(?P<ordinal>[0-9]+)"
+        "/page_interaction_email"
```

Comment:
Can you make that `page-interaction-email`? Just because all the other URL bits use dashes.